README.md

```diff
@@ -93,6 +93,20 @@ DatePickerDialog dpd = DatePickerDialog.newInstance(
 dpd.show(getFragmentManager(), "Datepickerdialog");
 ```
 
+You can also give from-to date by using following constructor:
+
+```java
+DatePickerDialog dpd = com.borax12.materialdaterangepicker.date.DatePickerDialog.newInstance(
+                        AnalyticsFragment.this,
```

Comment:
Please remove any references to your own class names like AnalyticsFragment.this